### PR TITLE
Add more data for CC load/store build operations

### PR DESCRIPTION
**Pull request by wolfs**
The PR adds the `cacheEntrySize` for CC load and store operation results, and the `originBuildInvocationId` for the load operation result.

### CODE REVIEW

**Review by facewindu — APPROVED** [review]

File: platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildOperationsIntegrationTest.groovy [hunk 1]
```diff
@@ -61,13 +61,24 @@ class ConfigurationCacheBuildOperationsIntegrationTest extends AbstractConfigura
...
+        }
+        with(loadOp.result) {
+            cacheEntrySize == storeOp.result.cacheEntrySize
+        }
+        def buildInvocationId = loadOp.result.originBuildInvocationId
 
         when:
         inDirectory 'lib'
         configurationCacheRun 'assemble'
+        loadOp = operations.only(ConfigurationCacheLoadBuildOperationType)
```
Comment (bamboo) [inline platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildOperationsIntegrationTest.groovy:77]: 💄 Minor thing, I'd declare a new variable here

File: platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt [hunk 2]
```diff
@@ -57,9 +67,12 @@ class ConfigurationCacheBuildOperationsTest {
         val buildOperationExecutor = mock<BuildOperationExecutor> {
             on { call<Unit>(any()) } doReturn Unit
         }
+        val stateFile = testDirectoryProvider.file("stateFile")
+        stateFile.text = "CC content"
```
Comment (bamboo) [inline platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt:71]: 🤔 Is the content needed?

File: platforms/enterprise/enterprise-operations/src/main/java/org/gradle/internal/configurationcache/ConfigurationCacheLoadBuildOperationType.java [hunk 3]
```diff
@@ -29,6 +31,22 @@ public interface Details {
...
+         * The number of bytes of the stored configuration cache entry.
+         *
+         * @since 8.6
+         */
+        long getCacheEntrySize();
+
+        /**
+         * The ID of the build that store the configuration cache entry.
+         *
+         * Is currently `null` when unknown, e.g. when loading models and not a task graph.
```
Comment (bamboo) [inline platforms/enterprise/enterprise-operations/src/main/java/org/gradle/internal/configurationcache/ConfigurationCacheLoadBuildOperationType.java:44]:
```suggestion
         * `null` when unknown, e.g. when loading models and not a task graph.
```

**Review by bamboo — APPROVED** [review]
🎉

File: platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt (hunk 2 above)
Comment (wolfs) [reply platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt:71]: Nope, it is not. Shall I remove it?

File: platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildOperationsIntegrationTest.groovy (hunk 1 above)
Comment (wolfs) [reply platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildOperationsIntegrationTest.groovy]: Done.

File: platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt (hunk 2 above)
Comment (wolfs) [reply platforms/core-configuration/configuration-cache/src/test/kotlin/org/gradle/configurationcache/ConfigurationCacheBuildOperationsTest.kt]: Removed.

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**